src/chat/chat_panel.rs

```diff
@@ -538,7 +541,11 @@ impl WidgetMatchEvent for ChatPanel {
                     self.redraw(cx);
                 }
                 ChatLineAction::Edit(id, updated, regenerate) => {
-                    store.chats.edit_chat_message(id, updated, regenerate);
+                    if regenerate {
```

Comment:
The flag was completely changing what the function does, so I spitted the function (motivated by [this](https://martinfowler.com/bliki/FlagArgument.html)).